### PR TITLE
Fixed error downloading summary files for some .sdfs 

### PR DESCRIPTION
**Pull request by hhan-schrodinger**
**Please review ACAS-301 before this PR**

## Description
There was an issue in the CReg bulk loader where summary files for some .sdfs (with special characters) could not be downloaded before and after registration. Added URI encoding in file download links to fix. 

## How Has This Been Tested?
Tested using an .sdf file with special characters (model 310522#1 (1).sdf), was able to download summary files before and after registration. Also was able to download original .sdf, current .sdf, and summary files from the Purge Files page in the CReg bulk loader.

### CODE REVIEW

File: modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee [hunk 1]
```diff
@@ -956,7 +956,9 @@ class FileRowSummaryController extends Backbone.View
 			currentFileName: currentFileName
 			fileLink: window.conf.datafiles.downloadurl.prefix + "cmpdreg_bulkload/" + fileName
```
Comment (bffrost) [inline modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee:957]: Is `fileLink` in here twice? Looks like it

**Review by bffrost — CHANGES_REQUESTED** [review]
Noticed a duplicate line. Other than that, looks good.

**Review by bffrost — APPROVED** [review]
LGTM!

File: modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee (hunk 1 above)
Comment (hhan-schrodinger) [reply modules/CmpdRegBulkLoader/src/client/CmpdRegBulkLoader.coffee]: @bffrost Good catch, thanks! Just fixed it.

**Pull request merged**